app/assets/stylesheets/in_favor_against.scss

```diff
@@ -22,6 +22,7 @@
     line-height: rem-calc(30);
     padding: rem-calc(3) rem-calc(6) rem-calc(6);
     position: relative;
+    opacity: 0.3;
```

Comment:
> For now, I'd keep the no-voted HTML class so it's possible to style buttons in this situation.

Come to think about it, maybe we don't need the `no-voted` class anyway, since now the `:has` selector is supported by 87.5% of the browsers and the styles for this case wouldn't be critical :thinking:. If we wanted to style the other button, we could do something like:

```scss
.in-favor-against {
  // (...)
  &:has([aria-pressed=true]) {
    [aria-pressed=false] {
      // Styles here
    }
  }
}
```

I'm fine with simply removing the `opacity: 0.3` rule, though, since IMHO makes the button look like a disabled button and it's confusing.